src/plot_utils.py

```diff
@@ -321,7 +318,7 @@ def guess_column_type(df_column):
         return guess_column_type(df_column)
 
 
-def conv_categorical_vals_to_numeric(data_srs, bdb=None, generator_name=None):
+def conv_categorical_vals_to_numeric(data_srs):
     # TODO: get real valuemap from btable
```

Comment:
Do you know what the `TODO` is referring to? It seems related to the unused arguments.